Bottom: 4a7126dc513292d00c013bf1eae328117eb12710
Top:    6bf0c9c3912cdb60e12cc89a479195bd74f8cd1f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-16 13:04:34 -0400

Refresh of efika-stac9766

---

diff --git a/sound/soc/fsl/efika-stac9766.c b/sound/soc/fsl/efika-stac9766.c
index 1cc2f0f..2dc06e1 100644
--- a/sound/soc/fsl/efika-stac9766.c
+++ b/sound/soc/fsl/efika-stac9766.c
@@ -130,7 +130,7 @@ static const struct snd_soc_dapm_widget efika_dapm_widgets[] = {
 };
 
 /* efika audio map */
-static const char *audio_map[][3] = {
+static const struct snd_soc_dapm_route audio_map[] = {
 
 	/* headphone connected to HPOUTL, HPOUTR */
 	{"Headphone Jack", NULL, "HPOUTL"},
